src/Core/Domain/Order/QueryResult/OrderForViewing.php

```diff
@@ -224,6 +230,7 @@ public function __construct(
         bool $isDelivered,
         bool $isShipped,
         bool $invoiceManagementIsEnabled,
+        string $note,
```

Comment:
@ks129 Could you add note at the end of parameters ? Else it's a BC to change signature.